libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/util/test/JSTestCase.java

```diff
@@ -72,9 +72,9 @@ public void setUp() throws Exception {
 
 	        // Wait for app initialization to complete
 			EventsListenerQueue eq = new EventsListenerQueue();
-	        if (SalesforceSDKManager.getInstance() == null) {
```

Comment:
getInstance throws an exception if init had not yet been called, so having the waitForEvent was pointless.
Now we do a safe check (by calling the new method hasInstance) and then wait if the app has not yet initialized.